### PR TITLE
Disable epoch fetcher

### PR DESCRIPTION
**Pull request by shazarre**

### CODE REVIEW

File: apps/indexer/lib/indexer/supervisor.ex [hunk 1]
```diff
@@ -188,12 +186,7 @@ defmodule Indexer.Supervisor do
 
     fetchers_with_metrics =
       if metrics_enabled do
-        metrics_processes = [
```
Comment (rkachowski) [inline apps/indexer/lib/indexer/supervisor.ex:191]: we need this patch here

**Review by rkachowski — APPROVED** [review]

File: apps/indexer/lib/indexer/supervisor.ex [hunk 2]
```diff
@@ -198,8 +196,19 @@ defmodule Indexer.Supervisor do
         fetchers_with_amb_bridge_mediators
       end
 
+    fetcher_with_epoch_rewards =
+      if System.get_env("DISPLAY_REWARDS") === "true" do
```
Comment (lvpeschke) [inline apps/indexer/lib/indexer/supervisor.ex:200]: Why System instead of Application here?

Comment (lvpeschke) [inline apps/indexer/lib/indexer/supervisor.ex:200]: Also the syntax looks different to other places where env variables are fetched, what's the difference with this one?

Comment (shazarre) [reply apps/indexer/lib/indexer/supervisor.ex:200]: Only reason is that I wanted to have exactly same check in both places: https://github.com/celo-org/blockscout/blob/master/apps/block_scout_web/lib/block_scout_web/templates/address/_tabs.html.eex#L70

and it's temporary anyway to exclude the possibility of this fetcher having such impact on the app

Comment (shazarre) [reply apps/indexer/lib/indexer/supervisor.ex:200]: same story as https://github.com/celo-org/blockscout/pull/705#discussion_r928673766 ;-)

**Pull request merged**